### PR TITLE
Reworked some notebooks

### PR DESCRIPTION
**Pull request by DerFerdi**
added the "important notes" to group 1 and 2,
added the ###start and ###end comments to group 1 and 2,
translated all variable names of group 1 to german,
reworked group_1_level_3`s definition of task,
fixed some typos,
removed some outputs,
fixed some errors

### CODE REVIEW

**Comment by DerFerdi** [comment]
I only just saw that you already finished reworking level 3 completely. I looked into those and think they are very good. 
But i still think you can use some of my other commits. There are some fixes for level 1 and 2, which i think are important. And also the commits for group 2 are no complete reworks but are a step in the right direction that will make reworking them easier.

File: notebooks/30_conditionals/exercises/group_1_level_1_25.ipynb [hunk 1]
```diff
@@ -17,21 +17,28 @@
...
+    "Handelsware beschafft werden soll. Soll die Handelsware aufgrund der oben genannten Bedingungen aktuell nicht\n",
     "beschafft werden soll die `bestellmenge` auf den Wert `0` gesetzt werden. \n",
     "\n",
+    "## Wichtige Hinweise\n",
+    "Bisher haben Sie im Rahmen dieser Einführung in Python Funktionen noch nicht kennen gelernt. Aus diesem Grund hier ein paar Hinweise zum Umgang mit Funktionen im Rahmen dieser Übungsaufgaben:\n",
+    "\n",
+    "1. Erstellen Sie Ihre Lösung innerhalb des mit `### START` und `### ENDE` markierten Bereichs. Ändern Sie den Programmcode außerhalb des markierten Bereichs nicht. \n",
+    "1. Funktionen besitzen Parameter. Diese Parameter haben Namen. Behandeln Sie die Parameter einfach wie Variablen in Ihrer Lösung. \n",
+    "1. Bitte löschen Sie ggf die Zeile `raise NotImplementedError()`\n",
+    "\n",
```
Comment (ceedee666) [inline notebooks/30_conditionals/exercises/group_1_level_1_25.ipynb:41]: Das hier scheint doppelt zu sein. Das steht doch danach auch wieder, oder?

**Review by ceedee666 — COMMENTED** [review]
Yes, I'll try to merge the branches and check will check what to keep.

File: notebooks/30_conditionals/exercises/group_1_level_1_25.ipynb (hunk 1 above)
Comment (DerFerdi) [reply notebooks/30_conditionals/exercises/group_1_level_1_25.ipynb:41]: oh ja, da habe ich mich wohl vertan

**Pull request merged**